instructor/patch.py

```diff
@@ -173,11 +175,28 @@ def handle_response_model(
             # if it is, system append the schema to the end
             else:
                 new_kwargs["messages"][0]["content"] += f"\n\n{message}"
+        elif mode == Mode.ANTHROPIC_TOOLS:            
+            tool_descriptions = json_to_xml(response_model.schema_json())
+            system_prompt = f"""In this environment you have access to a set of tools you can use to answer the user's question.
```

Comment:
you can't assume a system prompt does not already exist

Reply:
but also is the system prompt also actually 

create(system=str)

Reply:
> but also is the system prompt also actually
> 
> create(system=str)

yep, https://docs.anthropic.com/claude/docs/system-prompts#:~:text=system%3D%22Respond%20only%20in%20Spanish.%22%2C%20%23%20%3C%2D%2D%20system%20prompt